src/R/Editor/Impl/Completion/Definitions/RCompletionContext.cs

```diff
@@ -118,28 +96,22 @@ public static bool IsCaretInLibraryStatement(ITextView textView)
                     end -= line.Start;
 
                     string s = line.GetText().Substring(start, end - start);
-                    if (s == "library" || s == "require")
-                    {
+                    if (s == "library" || s == "require") {
                         return true;
                     }
                 }
-            }
-            catch (Exception) { }
-        
+            } catch (Exception) { }
+
             return false;
         }
 
-        public static string GetVariableName(ITextView textView, ITextSnapshot snapshot)
-        {
-             SnapshotPoint? pt = REditorDocument.MapCaretPositionFromView(textView);
-            if (pt.HasValue && pt.Value > 0)
-            {
+        public static string GetVariableName(ITextView textView, ITextSnapshot snapshot) {
+            SnapshotPoint? pt = REditorDocument.MapCaretPositionFromView(textView);
+            if (pt.HasValue && pt.Value > 0) {
                 int i = pt.Value - 1;
-                for (; i >= 0; i--)
-                {
+                for (; i >= 0; i--) {
                     char ch = snapshot[i];
-                    if (!RTokenizer.IsIdentifierCharacter(ch) && ch != '$' && ch != '@')
-                    {
+                    if (!RTokenizer.IsIdentifierCharacter(ch) && ch != '$' && ch != '@') {
```

Comment:
Nope. Code needs to be generally inspected for backtick handling. I am pretty sure there are quite a few places that may not be doing it right. Opened https://github.com/Microsoft/RTVS/issues/1360